js/test/index.spec.js

```diff
@@ -897,6 +897,18 @@ describe('index', () => {
         });
     });
 
+    it('should return all secrets starts with "some.secret"', () => {
+      const credstash = defCredstash();
+      return credstash.getAllSecrets({ startsWith: 'some.secret' })
+        .then((res) => {
+          Object.keys(res).length.should.equal(1);
+          Object.keys(res)[0].startsWith('some.secret');
```

Comment:
This needs to use .should, otherwise it is just evaluating the condition .startsWith and doing nothing with it.  
`Object.keys(res)[0].should.startWith('some.secret')`